src/components/Docs/SnippetViewer/ListObjectsRequestViewer.tsx

```diff
@@ -0,0 +1,133 @@
+    case SupportedLanguage.CURL:
+      /* eslint-disable max-len */
+      return `curl -X POST $FGA_API_URL/stores/$FGA_STORE_ID/list-objects \\
+  -H "Authorization: Bearer $FGA_BEARER_TOKEN" \\ # Not needed if service does not require authorization
+  -H "content-type: application/json" \\
+  -d '{
+        "type": "${objectType}",
+        "relation": "${relation}",
+        "user":"${user}"${
+        contextualTuples
+          ? `,
+        "contextual_tuples": {
+          "tuple_keys": [${contextualTuples
+              .map((tuple) => `
+            {"object": "${tuple.object}", "relation": "${tuple.relation}", "user": "${tuple.user}"}`)
+              .join(',')}
+          ]
+        }
+      }'`
+          : `
+      }'`
+        }
+
+# Response: {"object_ids": ["otherdoc", "planning"]}`;
+    /* eslint-enable max-len */
+    case SupportedLanguage.JS_SDK:
+      return `const response = await fgaClient.listObjects({${authorizationModelId ? `
+  authorization_model_id: "${authorizationModelId}",` : ``}
+  user: "${user}",
+  relation: "${relation}",
+  type: "${objectType}",${contextualTuples?.length ? `
+  contextual_tuples: {
+    tuple_keys: [${contextualTuples.map(tupleKey => `{
+      user: "${tupleKey.user}",
+      relation: "${tupleKey.relation}",
+      object: "${tupleKey.object}"
+    }`).join(', ')}]
+  },` : '' }
+});
+// response.object_ids = ["otherdoc", "planning"]`;
+    case SupportedLanguage.GO_SDK:
+      /* eslint-disable no-tabs */
+      return `body := fgaSdk.ListObjectsRequest{${authorizationModelId ? `
+    AuthorizationModelId: PtrString("${authorizationModelId}"),` : ``}
+    User:                 PtrString("${user}"),
+    Relation:             PtrString("${relation}"),
+    Type:                 PtrString("${objectType}"),${contextualTuples?.length ? `
+    ContextualTuples: &ContextualTupleKeys{
+        TupleKeys: []TupleKey{${contextualTuples.map(tupleKey => `{
+            User:     PtrString("${tupleKey.user}"),
+            Relation: PtrString("${tupleKey.relation}"),
+            Object:   PtrString("${tupleKey.object}"),
+        }`).join(', ')},
+    },` : '' }
+}
+
+data, response, err := apiClient.${languageMappings['go'].apiName}.ListObjects(context.Background()).Body(body).Execute()
+
+// data = { "object_ids": ["otherdoc", "planning"] }`;
+    case SupportedLanguage.DOTNET_SDK:
+      return `var body = new ListObjectsRequest{${authorizationModelId ? `
+    AuthorizationModelId = "${authorizationModelId}",` : ``}
+    User = "${user}",
+    Relation = "${relation}",
+    Type = "${objectType}",${contextualTuples?.length ? `
+    ContextualTuples = new ContextualTupleKeys() {
+        TupleKeys = new List<TupleKey> {${contextualTuples.map(tupleKey => `
+            new("${tupleKey.object}", "${tupleKey.relation}", "${tupleKey.user}")`).join(',')}
+        }
+    }` : '' }
+};
+var response = await openFgaApi.ListObjects(body);
+
+// response.ObjectIds = ["otherdoc", "planning"]`;
```

Comment:
These should be passed in as arguments from the caller, similar to the rest of the arguments.

If this component is to be used in a tutorial, the result needs to be in context

Reply:
added